templates/includes/nav.haml

```diff
@@ -259,6 +259,12 @@
       .title
         -trans "sign in"
 
+    %a{href:'{% url "oidc:openid_connect_login" auth_server="msft" %}'}
+      .circle
+        .icon-user
+      .title
+        -trans "sign in with microsoft"
```

Comment:
This is for microsoft's OpenID connect login alone ama?

Reply:
Yes, it is.

Reply:
_This is just a temporary change. For a deployment we can override this template and change the url for the main 'Sign in' button_